### PR TITLE
OFMCC-6477 - Create a Payments and Funding Request

### PR DESCRIPTION
**Pull request by Trevor-cgi**
Implements 
[https://eccbc.atlassian.net/browse/OFMCC-6477](https://eccbc.atlassian.net/browse/OFMCC-6477)

### CODE REVIEW

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalCreateFundingAndPaymentsRequest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+
+
```
Comment (Panika-cgi) [inline selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalCreateFundingAndPaymentsRequest.java:12]: extra lines of space

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/tests/portal/PortalPaymentsAndFundingRequest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    WebDriver driver;
+
+    @BeforeTest
+    public void initDriver() {
+        ChromeOptions options = new ChromeOptions();
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver(options);
+        driver.manage().window().maximize();
+    }
+
```
Comment (Panika-cgi) [inline selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/tests/portal/PortalPaymentsAndFundingRequest.java:24]: extra lines of space

Comment (Trevor-cgi) [reply selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/tests/portal/PortalPaymentsAndFundingRequest.java:24]: Got it! Done!

File: selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalCreateFundingAndPaymentsRequest.java (hunk 1 above)
Comment (Trevor-cgi) [reply selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalCreateFundingAndPaymentsRequest.java]: Done!

**Review by Panika-cgi — APPROVED** [review]
Looks great!

**Pull request merged**